### DISCUSSION TITLE
Integration with Woocommerce abandoned cart plugins OR Mailchimp

### DISCUSSION THREAD

**Discussion by parvnoor** [woocommerce/woocommerce#45094] [WooCommerce, Blocks, and Site Editing]
## Is your feature request related to a problem? Please describe.
Although the UX is much better for the woocommerce blocks, some other essential e-commerce tools aren't possible with the current blocks. Namely, the mailchimp + other abandon cart email collections don't call when someone abandons cart.

## Describe the solution you'd like
The ability for mailchimp + other cart abandonment trackers to track when user enters email and leaves cart without ordering.

**Comment by senadir** [comment]
Hey, integrating Mailchimp and other abandoned cart plugins (like AutomateWoo) is in our current agenda and it's a work on progress, so we should have answers around this in the near future (couple of months or so).

From my testing, AutomateWoo abandoned cart seems to work fine for logged in users, but not guests, if that's any consolation.

Thank you for your feedback, given that we have an issue tracking the status of MailChimp, I'm going to close this issue for now, feel free to comment back if you have any other question.

Have a great day!

**Comment by parvnoor** [comment]
Hi guys,

Wondering if there had been any progress or updates on this? 

Thanks!

**Comment by thisissandip** [comment]
Hey @parvnoor,

The MailChimp for WooCommerce is now compatible with the Cart and Checkout Blocks.

The minimum requirements are:

> WooCommerce Blocks 8.0.0+
Mailchimp for WooCommerce 2.7+

You can read and find other compatible extensions with Cart & Checkout blocks here: https://woocommerce.com/document/cart-checkout-blocks-support-status/#section-5

Should you have any questions, feel free to reach out to us and we will be happy to assist you! 🙂
